server/app/src/androidTest/java/sh/calaba/instrumentationbackend/utils/StringUtils.java

```diff
@@ -0,0 +1,17 @@
+package sh.calaba.instrumentationbackend.utils;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public final class StringUtils {
+
+    private StringUtils() {}
```

Comment:
Why do we need this empty `ctor`?

Reply:
This class is supposed to contain only static methods so we don't want anyone to create an instances of this class. By default, java creates an empty public constructor so instances are allowed to be created. To prevent it we create a private default constructor with overrides a default one. 
Honestly It's not necessary at all :) Just a small java-related hack